### ISSUE TITLE
Interactive merge request feature is broken in latest vscode

### ISSUE THREAD

**Issue by acoutts** [Dart-Code/Dart-Code#4109]
**Describe the bug**
The interactive debugging experience is broken on the latest vscode in dart files.

**To Reproduce**
1. Start a merge with merge conflicts
2. Open one of the conflict files in VSCode
3. Observe the missing interactive merge

**Expected behavior**
Should show blue/green sections to interactively take the head or incoming change. Instead you have to manually merge now.

**Screenshots**
<img width="940" alt="Screen Shot 2022-08-18 at 5 32 12 PM" src="https://user-images.githubusercontent.com/109968136/185498945-ba293809-86c8-4367-b1dc-2997aebd4e08.png">

**Please complete the following information:**
 - Operating System and version: macOS 12.5
 - VS Code version: 1.70.2 (stable, not insiders)
 - Dart extension version: 3.46.1
 - Dart/Flutter SDK version: 3.0.0 (stable channel)
 - Target device (if the issue relates to Flutter debugging): n/a

**Comment by DNAromance** [comment]
Same experience

**Comment by DanTup** [comment]
This appears to be VS Code behaviour. A new setting "Git: Merge Editor" is enabled by default, and it prevents the original CodeLens merging from working. Disabling the setting reverts this, but it seems odd to me that this is intended behaviour (I don't see any new merge editor, and if that's because I've disabled built-in Git, then why does it still disable the original behaviour?).

I've filed https://github.com/microsoft/vscode/issues/158763 to enquire whether this is intended or not, but for now I'd suggest just disabling "Git: Merge Editor".
